src/BrandedNavBar/SubMenuTrigger.tsx

```diff
@@ -15,36 +15,13 @@ type SubMenuTriggerProps = React.ComponentPropsWithRef<"button"> & {
   menuData: any[];
 };
 
-const StyledButton: StyledComponentPropsWithRef<any> = styled.button(
-  ({ isOpen }: any): CSSObject => ({
-    display: "block",
+const StyledButton: StyledComponentPropsWithRef<any> = styled(DropdownButton)(
+  ({ isOpen, theme }: any): CSSObject => ({
     position: "relative",
-    color: theme.colors.darkBlue,
-    fontSize: theme.fontSizes.small,
-    lineHeight: theme.lineHeights.smallTextBase,
-    width: "100%",
-    padding: `${theme.space.half} 28px ${theme.space.half} 12px`,
-    border: "none",
-    borderLeft: `${theme.space.half} solid transparent`,
-    "&:hover": {
-      backgroundColor: theme.colors.lightGrey,
-    },
-    "&:disabled": {
-      opacity: ".5",
-    },
-    "&:focus": {
-      outline: "none",
-      color: theme.colors.darkBlue,
-      backgroundColor: theme.colors.lightGrey,
-      borderLeft: `${theme.space.half}  solid ${theme.colors.blue}`,
-    },
-    backgroundColor: isOpen ? theme.colors.lightGrey : "transparent",
-    textDecoration: "none",
-    textAlign: "left",
-    cursor: "pointer",
+    backgroundColor: isOpen ? theme.colors.lightBlue : "transparent",
+    color: isOpen ? theme.colors.darkBlue : theme.colors.darkGrey,
```

Comment:
@haideralsh is there a better way of pulling these colour values from the `DropdownButton`? Or getting the defaults somehow?

Reply:
Ok, I'm gonna leave it be then.